playbooks/storage.yml

```diff
@@ -21,3 +21,13 @@
   roles:
     - role: storage-demo-nodeconfig
       when: storage_role == 'storage-demo'
+
+- hosts: localhost
+  vars:
+  gather_facts: False
+  # unset http_proxy. required for running in the CI
+  environment:
+    http_proxy: ""
+  roles:
+    - roles: storage-cinder
```

Comment:
does cinder need the cdi role?

Reply:
Yes.  All storage flavors should include it.